source/partials/_changelog.erb

```diff
@@ -0,0 +1,37 @@
+<!-- Change log -->
+
+<h2 class="govuk-heading-m">Documentation updates</h2>
+
+<p>These are the most recent changes to this documentation.</p>
+
+<table class="govuk-table">
```

Comment:
I believe we format our tables differently to this - see other pages for reference, e.g. https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/prove-users-identity/#understand-your-user-s-address-claim. Makes it easier to update and parse what information is included.

Reply:
@PippaClarkGDS have converted the html and design system into plain markdown as suggested